packages/terra-clinical-data-grid/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "keycode-js": "^1.0.4",
+    "keycode-js": "^2.0.1",
```

Comment:
keycode-js 3.x is out. Should we go straight to that?

Reply:
https://www.npmjs.com/package/keycode-js

Reply:
Updated to 3.x:
https://github.com/cerner/terra-clinical/pull/730/commits/2aac14f19512ab431f1c804266202937f89a6788
https://github.com/cerner/terra-clinical/pull/730/commits/33994fb66a5135eec549530a32f367cf9bfcf32b